src/style.css

```diff
@@ -0,0 +1,111 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  padding: 30px 14px 14px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__photo {
+  width: 160px;
+  height: 134px;
+}
+
+.card__description {
+  width: 170px;
+  margin-top: 45px;
+  margin-bottom: 20px;
+}
+
+.card__name {
+  margin: 0;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+}
+
+.card__code {
+  display: block;
+  margin-top: 6px;
+}
+
+.card__caption,
+.card__code,
+.card__feedback {
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.card__rate {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 14px;
+}
+
+.card__stars {
+  display: flex;
+  justify-items: center;
+  align-items: center;
+}
+
+.card__feedback {
+  align-self: flex-end;
+}
+
+.stars__item {
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+  background: url(images/star.svg) no-repeat center;
+}
+
+.card__value {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-top: 19px;
+}
+
+.card__price {
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 19px;
+}
+
+.stars__item:nth-child(-n+4) {
```

Comment:
add classes from stars block task, so if I change the class of your element in dev tools I will see how the count of active stars changing.